config/initializers/prx_auth.rb

```diff
@@ -0,0 +1,5 @@
+require 'prx_auth/rails'
+
+PrxAuth::Rails.configure do |config|
+  config.install_middleware = ENV['ID_HOST'].blank?
```

Comment:
No feeder namespace here?  Oh, I guess `PrxAuth::Rails::Configuration.initialize` will guess the namespace from your rails app.  Does that not work over in CMS or something?

Also I'm still a little confused why you only install the middleware if ID_HOST is blank - shouldn't it be the opposite?

Reply:
the cms application is called Prx, so I use the namespace option to make it :cms instead of :prx - :feeder is already inferred here, but I can make it explicit if you prefer.

I answered the second question in cms, but short version is that we disable middleware installation when we are installing our own version to manually set the ID host

Reply:
Makes sense!  +1